geps/gep-91.md

```diff
@@ -16,7 +16,71 @@ This GEP proposes a way to validate the TLS certificate presented by the downstr
 ## Non-Goals
 - Define other fields that can be used to verify the client certificate such as the Cerificate Hash or Subject Alt Name. 
 
+### API
```

Comment:
I'd like to know for which route types (HTTP, TLS, TCP, UDP, GRPC, etc) , protocols, and termination types (edge, passthrough?) this might be applicable?

Reply:
this is same as the downstream info mentioned in https://gateway-api.sigs.k8s.io/guides/tls/#clientserver-and-tls, does this need to be added in the GEP ?